src/components/checkboxFilter/index.tsx

```diff
@@ -53,16 +55,34 @@ function CheckboxFilter<ItemKey extends string>({
             key={`filter_${name}_${item.label}`}
             name={`filter_${name}_${item.label}`}
             label={
-              <chakra.span
-                w="full"
-                display="flex"
-                justifyContent="space-between"
-              >
-                <chakra.span>{item.label}</chakra.span>
-                <chakra.span>
-                  {addThousandSeparatorToNumber(item.facet)}
+              item.image ? (
+                <chakra.span display="flex" alignItems="center">
+                  {item.image}
+                  <chakra.span
+                    w="full"
+                    display="flex"
+                    justifyContent="space-between"
+                    flexWrap="wrap"
+                    marginLeft="sm"
+                  >
+                    <chakra.span>{item.label}</chakra.span>
+                    <chakra.span>
```

Comment:
As far as I am aware, the gray version of the facet was the most recent one. Maybe you can quickly check with one of the designers

Reply:
As we checked in the refinement. We will keep it as it is in preprod -> dark gray. And if the UX-designers want a change, they can bring it.